docs/index.html

```diff
@@ -275,23 +310,41 @@ <h2>#TwitterThrowback</h2>
             loadingIndicator.style.display = 'block';
         }, 1000);
     }
+
+    function shareToTwitter() {
+        let url;
+        if(window.username !== undefined) {
+            url = `http://twitter.com/share?text=Here's my TwitterThrowback. Check yours here` +
```

Comment:
Make it a hashtag. "Here's my #TwitterThrowback." You might need to use `&39;` for that .

Oh, never mind. Saw the Twitter option for that.

Reply:
I shouldn't bother with &39; ?